featuretoggles/enablement_level.go

```diff
@@ -56,7 +56,7 @@ func ComputeEnablementLevel(ctx context.Context, feature *unleashapi.Feature, in
 	return fromFeatureLevel(enablementLevel)
 }
 
-func toFeatureLevel(level string, internalUser bool) FeatureLevel {
+func ToFeatureLevel(level string, internalUser bool) FeatureLevel {
```

Comment:
there's no need to make this function public, is there ? The `whitebox` test for this function is in the same package, so it can already call this function.